x-pack/plugins/security_solution/public/management/components/endpoint_responder/action_log_button.tsx

```diff
@@ -30,7 +30,7 @@ export const ActionLogButton = memo<EndpointResponderExtensionComponentProps>((p
       >
         <FormattedMessage
           id="xpack.securitySolution.actionLogButton.label"
```

Comment:
You can also update the translation key here to something like `xpack.securitySolution.endpointResponder.responseActionsHistoryButton.label`

Reply:
updated